sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1830,6 +1830,13 @@ object SQLConf {
       .stringConf
       .createWithDefault("lz4")
 
+  val CHECKPOINT_RENAMEDFILE_CHECK_ENABLED =
+    buildConf("spark.sql.streaming.checkpoint.renamedFileCheck")
```

Comment:
I think maybe we usually "enabled" keyword for string when "ENABLED" is following in the postfix for option?

So, for example:
```diff
-  "spark.sql.streaming.checkpoint.renamedFileCheck"
+  "spark.sql.streaming.checkpoint.renamedFileCheck.enabled"
```

WDYT?

Reply:
Okay. LGTM.